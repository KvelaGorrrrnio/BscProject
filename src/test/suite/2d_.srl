list  list  list  int S
list  list  int S1
list  int S2
int n
list  list  int p
list  int q
int s

from (? S) do
  from (? S1) do
    from (? S2) do
      push s S2
    loop
      skip
    until (# S2 = 3)
    push S2 S1
  loop
    skip
  until (# S1 = 1)
  push S1 S
loop
  skip
until (# S = 1)
