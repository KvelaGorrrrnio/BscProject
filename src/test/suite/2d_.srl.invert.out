list  list  list  int S
list  list  int S1
list  int S2
int n
list  list  int p
list  int q
int s

from (# S = 1) do
  pop S1 S
  from (# S1 = 1) do
    pop S2 S1
    from (# S2 = 3) do
      pop s S2
    loop
      skip
    until (? S2)
  loop
    skip
  until (? S1)
loop
  skip
until (? S)
